physics/SFC_Models/Land/Noahmp/module_sf_noahmplsm.F90

```diff
@@ -5607,6 +5630,11 @@ subroutine gfs_stability                                              &
 real(kind=kind_phys) :: zolmax
 
 real(kind=kind_phys) xkzo
+!   tofd
```

Comment:
does this line need to be here? if so, be more descriptive

Reply:
removed